src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/personalrelationships/profiledetails/ContactPersonProfileDetailsReconciliationIntTest.kt

```diff
@@ -288,8 +295,11 @@ class ContactPersonProfileDetailsReconciliationIntTest(
         runReconciliation()
       }
 
-      @Test
-      fun `should run a reconciliation with no problems`() {
+      @ParameterizedTest
+      @ValueSource(longs = [7, 6])
```

Comment:
Most of the work on the tests has been to make the set up generic so we can test for both full and partial last pages.